index.js

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env node
+require("dotenv").config();
+const axios = require("axios");
+
+// Store Slack access token in a .env file in the root folder
+const accessToken = process.env.SLACK_ACCESS_TOKEN;
+
+async function getConversationId(conversation) {
```

Comment:
Can we rename Conversation to Channel? Conversation == Slack Channel, no?

Reply:
Mark just pointed out to me that there were comments here. Sorry, I never got notified via email. D'oh!

We absolutely could. The reason it is "Conversation" is because Slack is pushing towards using their Conversation API, which could be a channel or a private DM, etc. I named it line up with the API. But in this context, yes, it could just be channel.